### PR TITLE
#842 Ability to sort by multiple metrics

### LINKED ISSUE #842: Ability to sort by multiple metrics
Right now I would like to sort by long time no interaction and sort it from the most expensive.

### adjust existing filter names 
New First -> Recently Created
Old first -> Oldest
Last interacted -> Recently Interacted
Long time no interaction -> Unpopular
From most expensive -> Price: High to Low
From cheaper -> Price: Low to High

### Spec
- Add recently listed for sale
- It would require writing a new resolver to https://github.com/kodadot/rubick

image

image

### Ref
- probably ideation to add **min - max**  -> https://github.com/kodadot/nft-gallery/issues/2110

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #842
- [ ] 

### Before submitting Pull Request, please make sure:

- [ ] My contribution builds **clean without any errors or warnings**
- [ ] I've merged recent default branch -- **main** and I've no conflicts
- [ ] I've tried to respect high code quality standards
- [ ] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by prachi00** [comment]
So I am in still in doubt about the query
because if I sort by `most reacted` and `high to low` at the same time, the result is confusing, not sure how to fix it though. 

Secondly, there is this weird error on console even though I think I did it right , its happening because of having 2 VModels but they have different values so it should be fine right?
Still trying to figure this one out but thought I'd raise the PR just in case someone else has any idea

Screen Shot 2022-05-25 at 6 36 49 PM
 
 
Screen Shot 2022-05-25 at 6 37 31 PM

**Comment by prachi00** [comment]
> So I am in still in doubt about the query because if I sort by `most reacted` and `high to low` at the same time, the result is confusing, not sure how to fix it though.
> 
> Secondly, there is this weird error on console even though I think I did it right , its happening because of having 2 VModels but they have different values so it should be fine right? Still trying to figure this one out but thought I'd raise the PR just in case someone else has any idea
> 
> Screen Shot 2022-05-25 at 6 36 49 PM
> 
> Screen Shot 2022-05-25 at 6 37 31 PM

CC @vikiival @roiLeo

**Comment by prachi00** [comment]
> IMO I would rather use https://buefy.org/documentation/dropdown/#multiple :)

used dropdown component

**Review by vikiival — COMMENTED** [review]
code looks LGTM
would be nice to test it

File: components/rmrk/Gallery/Search/SearchSortDropdown.vue [hunk 1]
```diff
@@ -1,6 +1,19 @@
 <template>
   <b-field>
+    <b-dropdown
+      v-if="multipleSelect"
+      multiple
+      v-model="selectedActionArray"
+      class="select-dropdown">
+      <template #trigger>
+        <b-button type="is-primary" icon-right="menu-down"> Sort by </b-button>
```
Comment (roiLeo) [inline components/rmrk/Gallery/Search/SearchSortDropdown.vue:9]: Missing translation here + fix icon
```suggestion
        <b-button type="is-primary" icon-right="caret-down"> Sort by </b-button>
```

**Review by roiLeo — COMMENTED** [review]
Tested on `/rmrk/explore?page=1&tab=GALLERY&search=&sort=PRICE_DESC&sort=EMOTES_COUNT_DESC&listed=true`

**Review by petersopko — CHANGES_REQUESTED** [review]
@prachi00 there was a request to rename most of the filters in the issue, please check

**Comment by prachi00** [comment]
> @prachi00 there was a request to rename most of the filters in the issue, please check

Screen Shot 2022-06-02 at 5 13 16 PM
Done

**Review by petersopko — APPROVED** [review]
LGTM

**Comment by petersopko** [comment]
pay 200 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $200 @ 68.5 USD/KSM ~ 2.92 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0xd8fa62324483c87739eed855972c58880f7be00a20ba7ba810564c2de10feac5](https://kusama.subscan.io/extrinsic/0xd8fa62324483c87739eed855972c58880f7be00a20ba7ba810564c2de10feac5)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**